operator/plugin.go

```diff
@@ -63,6 +56,19 @@ func (r PluginRegistry) Render(pluginType string, params map[string]interface{})
 		)
 	}
 
+	for name, param := range config.Parameters {
+		if err := param.validate(); err != nil {
+			return PluginConfig{}, errors.NewError(
+				"invalid parameter description found in plugin config",
+				"ensure that all parameter descriptioins are valid for the plugin",
```

Comment:
Descriptions is misspelled. Also, this error message is a little confusing, since validate is checking several different aspects of the param, and not just the param description.

Reply:
Good catch. I was using descriptions to mean the overall definition of the parameter, but that is clearly not a good choice of terminology.